slatedb/src/mem_table_flush.rs

```diff
@@ -290,21 +289,25 @@ impl DbInner {
         let this = Arc::clone(self);
         Some(spawn_bg_task(
             tokio_handle,
-            move |result| {
-                let err = bg_task_result_into_err(result);
-                warn!("memtable flush task exited with error [error={}]", err);
-                let mut state = this.state.write();
-                state.record_fatal_error(err.clone());
-                info!("notifying in-memory memtable of error");
```

Comment:
same here, I believe beforehand we were notifying watchers about the shutdown but now we're just logging. Does this cause any issues?